catalog/util.sh

```diff
@@ -4,8 +4,9 @@
 #
 # Note use of --apihost, this is needed in case of a b/g swap since the router may not be
 # updated yet and there may be a breaking change in the API. All tests should go through edge.
-SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
-WHISKPROPS_FILE="$SCRIPTDIR/../whisk.properties"
+
+: ${OPENWHISK_HOME:?"OPENWHISK_HOME must be set and non-empty"}
```

Comment:
Note that rather than compute OPENWHISK_HOME relative to this file, I opted to enforce using the property instead for consistency with other scripts.